flang/test/Driver/main_dupes.f90

```diff
@@ -0,0 +1,13 @@
+! RUN: %clang -x c -o %t.c-part -c %s.c-part
```

Comment:
Could you try `flang-new -x c` instead? It would be great if we could avoid using the `clang` binary in Flang (to avoid increasing the dependency).

Reply:
That does not seem to work and gives me this error when running the test.

`error: unknown integrated tool '-cc1'. Valid tools include '-fc1'.`

I do not necessarily need `clang` as the C compiler for the test.  GCC (or whatever was used to build that instance of LLVM) would also be sufficient.

I guess that problem will come back when testing `BIND(C)`.  So, maybe a more generic and better solution might be needed.  But I guess that may be beyond this PR.

Reply:
That's a shame - thanks for checking. One alternative would be to add LLVM IR instead of a C file, but LLVM IR is not stable. Though a plain function definition should be fairly stable:
```
define i32 @main(i32 noundef %0, ptr noundef %1) {
  ret i32 0
}
```

Would this work? @kiranchandramohan - what do you think?

Reply:
That has worked.  I haven't thought about that at all, so many thanks for the suggestion.  This greatly simplifies things!